src/pybind/mgr/dashboard/frontend/src/app/ceph/block/mirroring/daemon-list/daemon-list.component.html

```diff
@@ -1,13 +1,11 @@
-<cd-table [data]="data"
-          columnMode="flex"
-          [columns]="columns"
-          [autoReload]="-1"
-          (fetchData)="refresh()"
-          [status]="tableStatus">
-</cd-table>
+<ng-container *ngIf="empty">
+  <strong class="text-danger"
+          i18n>There are no daemons</strong>
+</ng-container>
 
-<ng-template #healthTmpl
-             let-row="row"
-             let-value="value">
-  <span [ngClass]="row.health_color | mirrorHealthColor">{{ value }}</span>
-</ng-template>
+<ng-container *ngFor="let daemon of daemons | keyvalue">
+  <span *ngIf="daemon.value > 0"
+        [ngClass]="daemon.key | mirrorHealthColor"
+        class="m-1"
+        i18n>{{daemon.key | daemonStatus}}</span>
```

Comment:
Did you test this?

that is how a pipe is used but `daemonStatus` itself isn't a pipe, it is a function. 
![image](https://user-images.githubusercontent.com/30913090/174612423-4be7c493-0ca2-46ec-975e-1455e6693a46.png)
Check shared/pipes to see how they are implemented.
![image](https://user-images.githubusercontent.com/30913090/174612609-4157ada4-1165-4f76-8f0e-b68ca16889a8.png)

Reply:
We [have a pipe](https://www.npmjs.com/package/ngx-pipe-function) that allows to call any (pure) function.